### ISSUE TITLE
Port twisted/python/modules.py to use @implementer

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7155]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7155|
|Type|enhancement|
|Created|2014-04-15 16:54:35Z|
|Branch|https://github.com/twisted/twisted/tree/modules-implementer-7155|

Right now it uses implements, which doesn't support Python3.

<details><summary>Searchable metadata</summary>

```
trac-id__7155 7155
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_modules_implementer_7155 branches/modules-implementer-7155
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1397580875000000 1397580875000000
changetime__1397585389000000 1397585389000000
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

Patch is available at https://github.com/twisted/twisted/pull/12

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

I think you mean https://github.com/twisted/twisted/pull/12.patch ;-)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph commented|
|-|-|

A word to the wise: _always_ disable LastPass's "automatically fill login information" option.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph commented|
|-|-|

(In [42254]) Branching to modules-implementer-7155.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph set owner to @glyph|
|-|-|

Looks good, will land.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_8"></a>|@glyph set status to `closed`|
|-|-|

(In [42265]) Merge modules-implementer-7155.

Stop using implements() in one more place.

Author: Alex

Reviewer: glyph

Fixes: [#7155](https://github.com/twisted/twisted/issues/7155)
